README.md

```diff
@@ -256,6 +256,72 @@ to return forces.
 torch_force.setOutputsForces(True)
 ```
 
+Computing energy derivatives with respect to global parameters
+--------------------------------------------------------------
+
+Its possible to query `TorchForce` for the derivative of the energy with respect to global parameters. In order to do so the global parameters must be registered as energy derivatives. This is done by calling `addEnergyParameterDerivative()` for each parameter.
+
+The parameter derivatives can be queried by calling `getEnergyParameterDerivatives()` on the `State` object returned by `Context.getState()`. The result is a dictionary with the parameter names as keys and the derivatives as values.
+
+```python
+import torch as pt
+from torch import Tensor
+from openmmtorch import TorchForce
+import openmm as mm
+
+class ForceWithParameters(pt.nn.Module):
+
+    def __init__(self):
+        super(ForceWithParameters, self).__init__()
+
+    def forward(
+        self, positions: Tensor, parameter1: Tensor, parameter2: Tensor
+    ) -> Tensor:
+        x2 = positions.pow(2).sum(dim=1)
+        u_harmonic = ((parameter1 + parameter2**2) * x2).sum()
+        return u_harmonic
```

Comment:
This seems overly complicated for an example;  One parameter would illustrate the point as well as two.  And I don't quite follow your code.  You sum over one dimension, multiply by a constant, and then sum over the remaining dimensions?

How about this:

```python
def forward(self, positions: Tensor, k: Tensor) -> Tensor:
    return k*torch.sum(positions**2)
```

(which is exactly the same as the earlier example for global parameters).